Protacon.RxMq.AzureServiceBus.Tests/AzureBusQueueIntegrationTests.cs

```diff
@@ -26,7 +26,7 @@ public async void WhenMessageIsSend_ThenItCanBeReceived()
 
             await subscriber.Messages<TestMessage>()
                 .Where(x => x.ExampleId == id)
-                .Timeout(TimeSpan.FromSeconds(5))
+                .Timeout(TimeSpan.FromSeconds(30))
```

Comment:
Why these changes? Same on the next test too. Failing otherwise? What has changed that causes this?

Reply:
They are failing in devops that is slower than local